declarations_site/catalog/paginator.py

```diff
@@ -0,0 +1,79 @@
+from django.core.paginator import Paginator, Page
+from django.utils import six
+from django.conf import settings
+
+from elasticsearch_dsl.result import Response
+
+
+class ElasticPaginator(Paginator):
```

Comment:
They don't have contribs and I doubt they would want that. Maybe some Django wrapper like this instead:
https://github.com/Sparrho/bungiesearch